Add option to show/hide Templater ribbon icon

fmbarina opened the pull request:
Enhancement #647 I suggested recently.

Add a plugin option to show or hide the sidebar/ribbon icon. There are other ways, including even [another obsidian extension](https://github.com/phibr0/obsidian-customizable-sidebar) with an option to hide the Templater icon. I think it makes more sense and is better for the extension itself to offer that option.

Changes Made:
- Added boolean setting enable_ribbon_icon, default true, toggle option hides/shows icon.
- Added check to see if setting is true before adding icon in main.

shabegom reviewed (changes requested):
just needs one small language tweak

fmbarina commented:
Thanks, changed. Really makes me wonder if I can truly call myself fluent.
Squashed into a single commit so that it fits better in the commit history. I think that's how you're supposed to do it?

shabegom commented:
Sweet, thanks! Will merge and add to next release
